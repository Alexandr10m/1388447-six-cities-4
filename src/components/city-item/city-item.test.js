import React from "react";
import renderer from "react-test-renderer";
import CityItem from "./city-item.jsx";

describe(`Snapshot of CityItem`, () => {
  it(`CityItem should render correctly with active-class`, () => {
    const tree = renderer
      .create(
          <CityItem
            city={`Amsterdame`}
            currentCity={`Amsterdame`}
          />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`CityItem should render correctly without active-class`, () => {
    const tree = renderer
      .create(
          <CityItem
            city={`Paris`}
            currentCity={`Amsterdame`}
          />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
