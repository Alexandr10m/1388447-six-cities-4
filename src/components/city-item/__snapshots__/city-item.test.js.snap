// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of CityItem CityItem should render correctly with active-class 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="/Paris"
    onClick={[Function]}
  >
    <span>
      Paris
    </span>
  </a>
</li>
`;

exports[`Snapshot of CityItem CityItem should render correctly without active-class 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item "
    href="/Paris"
    onClick={[Function]}
  >
    <span>
      Paris
    </span>
  </a>
</li>
`;
